app/views/edit/build-config.html

```diff
@@ -472,7 +472,7 @@
                     <button
                       type="submit"
                       class="btn btn-primary btn-lg"
-                        ng-disabled="form.$invalid || form.$pristine || disableInputs">
+                        ng-disabled="form.$invalid || form.$pristine || disableInputs || isSourceURLInvalid()">
```

Comment:
We made this a warning rather than an error on the "add to project" page. Suggest the same here. It's a complicated regex, and in at least one case, we had it wrong. I don't think we should block users from submitting the form.